src/dbSta/src/dbSta.cc

```diff
@@ -536,17 +536,25 @@ void dbSta::report_cell_usage(odb::dbModule* module, const bool verbose)
                     module->getName());
   }
   logger_->report(header_format, "Cell type report:", "Count", "Area");
+
+  const std::regex regexp(" |/|-");
   for (auto [type, stats] : instances_types) {
-    std::string type_name = getInstanceTypeText(type);
+    const std::string type_name = getInstanceTypeText(type);
     logger_->report(
         format, type_name, stats.count, stats.area / area_to_microns);
     total_area += stats.area;
 
-    std::regex regexp(" |/|-");
-    logger_->metric("design__instance__count__class:"
-                        + toLowerCase(regex_replace(type_name, regexp, "_")),
+    const std::string type_class
+        = toLowerCase(regex_replace(type_name, regexp, "_"));
+
+    logger_->metric("design__instance__count__class:" + type_class,
                     stats.count);
+    logger_->metric("design__instance__area__class:" + type_class,
+                    stats.area / area_to_microns);
   }
+  logger_->metric("design__instance__count__class:total", total_usage);
+  logger_->metric("design__instance__area__class:total",
+                  total_area / area_to_microns);
```

Comment:
We already have design__instance__count, do we need design__instance__count__class:total?  Likewise design__instance__area and design__instance__area__class:total

Reply:
This provides the number for the selected modinst (design__instance__area is for the entire design)

Reply:
If you run report_cell_usage on two different modules then the metrics will collide.

Reply:
this is true to many commands, I change the metrics stage in my script before each call, but we could change the metrics prefix if you have a suggestion.

Reply:
for the top module would you drop the `__in_module`?

Reply:
The top module is the same as whole design, ie design__instance__count.